### PR TITLE
Adding ability to specify platform in `arcs_kt_schema` rule. 

### PR DESCRIPTION
**Pull request by alxrsngrtn**

### CODE REVIEW

**Comment by cromwellian** [comment]
I don't disagree with allowing this, but what motivated it? Some wasm issue?

**Comment by bayesianmind** [comment]
Yes we're having build issues internally related to wasm. Expedient thing for now is to turn it off in g3.

File: third_party/java/arcs/build_defs/internal/schemas.bzl [hunk 1]
```diff
@@ -57,19 +57,23 @@ def arcs_cc_schema(name, src, deps = [], out = None):
         wasm = False,
     )
 
-def arcs_kt_schema(name, srcs, deps = [], test_harness = True):
+def arcs_kt_schema(name, srcs, deps = [], platforms = ["jvm", "wasm"], test_harness = True):
```
Comment (piotrswigon) [inline third_party/java/arcs/build_defs/internal/schemas.bzl]: Given the status of wasm I would argue for making `["jvm"]` the default.

Comment (alxrsngrtn) [reply third_party/java/arcs/build_defs/internal/schemas.bzl]: > Given the state of wasm I would argue for making `["jvm"]` the default.

In applying this change I discovered a minor bug w/ our Bazel rules: 
If we have just "wasm" as a platform target, it fails to build the `arcs_kt_schema` rule. 

I added a workaround to always build the `jvm` target, even when it's not needed. The source of the problem is in how `arcs_kt_library` is defined.

**Comment by alxrsngrtn** [comment]
Skipping presubmit, going to sort it out in manual copybara.

**Pull request merged**